evap/staff/views.py

```diff
@@ -1418,9 +1423,15 @@ def helper_single_result_edit(request, evaluation):
 
 @require_POST
 @manager_required
+@transaction.atomic
 def evaluation_delete(request):
     evaluation = get_object_from_dict_pk_entry_or_logged_40x(Evaluation, request.POST, "evaluation_id")
 
+    # See comment in helper_evaluation_edit
+    @receiver(RewardPointGranting.granted_by_evaluation_deletion, weak=True)
+    def notify_reward_points(grantings, **_kwargs):
+        logger.info(f"Deletion of evaluation {evaluation} has created {len(grantings)} reward point grantings")
```

Comment:
@janno42 evaluation deletion currently just removes the row, it doesn't reload the page, so a `messages.info` call will only be shown on the next page (re)load. Do we want this to be shown to users, and if yes, what should it look like?